File: woodroid/src/main/java/me/gilo/woodroid/data/ApiVersion.java
package me.gilo.woodroid.data;

public enum ApiVersion {
    API_VERSION1{
        @Override
        public String toString() {
            return "1";
        }
    },
    API_VERSION2{
        @Override
        public String toString() {
            return "2";
        }
    },
    API_VERSION3{
        @Override
        public String toString() {
            return "3";
        }
    },
}

File: woodroid/src/main/java/me/gilo/woodroid/data/ApiVersion.kt
package me.gilo.woodroid.data

enum class ApiVersion {
    API_VERSION1 {
        override fun toString(): String {
            return "1"
        }
    },
    API_VERSION2 {
        override fun toString(): String {
            return "2"
        }
    },
    API_VERSION3 {
        override fun toString(): String {
            return "3"
        }
    }
}